test/unit/io_managers_test.py

```diff
@@ -359,161 +355,6 @@ def test_error_when_reading_view_without_metadata(fake_pudl_sqlite_io_manager_fi
         fake_pudl_sqlite_io_manager_fixture.load_input(input_context)
 
 
-def test_ferc_xbrl_sqlite_io_manager_dedupes(mocker, tmp_path):
```

Comment:
this got moved over into the transform unit test. but it also got significantly gutted bc we needed basically none of this complex setup to test it now